docs/api.rst

```diff
@@ -9340,6 +9340,118 @@ Editing blockchain account data
    :statuscode 409: An account given to edit does not exist or a given tag does not exist.
    :statuscode 500: Internal rotki error
 
+
+Chain agnostic account operations
+==================================
+
+.. http:patch:: /api/(version)/blockchains/agnostic/accounts
```

Comment:
Name. and format.

```suggestion
Account operations by chain type
==================================

.. http:patch:: /api/(version)/blockchains/(chaintype)/accounts
```

So make the chaintype be part of the URL string. Should only `evm`. `bitcoin` or `substrate`